Sources/OpenTelemetryInstrumentationSupport/SpanAttribute+GRPCSemantics.swift

```diff
@@ -0,0 +1,71 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Tracing open source project
+//
+// Copyright (c) 2020 Moritz Lang and the Swift Tracing project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import TracingInstrumentation
+
+extension SpanAttributeName {
+    /// - See: GRPCAttributes
```

Comment:
I think those need to be in `` for jazzy to make it into links?

We can do a proper documentation pass over all docs next week though.

Reply:
Yeah, let's do it next week when we go through everything 👀